instrumentation/dubbo-rpc/src/main/java/brave/dubbo/rpc/DubboClientResponse.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2013-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.dubbo.rpc;
+
+import brave.internal.Nullable;
+import brave.rpc.RpcClientResponse;
+import com.alibaba.dubbo.rpc.Result;
+import com.alibaba.dubbo.rpc.RpcException;
+
+class DubboClientResponse extends RpcClientResponse implements DubboResponse { // not final for mock
```

Comment:
May consider using mockito's support for mocking finals. I've been using it happily for a while. No idea how it works ;)

https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#unmockable

Reply:
great sorcery. I prefer to have oddness in src/test vs src/main any time!